internal/campaigns/service.go

```diff
@@ -312,7 +312,34 @@ query NamespaceQuery($name: String!) {
 }
 `
 
+const usernameQuery = `
+query {
```

Comment:
I think it's good practice to name your queries, but might be wrong:

```suggestion
query GetCurrentUserID {
```